policyengine_us/variables/gov/states/nm/tax/income/credits/nm_child_income_credit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class nm_child_income_credit(Variable):
```

Comment:
For consistency with other state CTCs
```suggestion
class nm_ctc(Variable):
```